src/Profiles.ts

```diff
@@ -149,12 +133,75 @@ export class Profiles {
             }
         });
         if (profileStatus === undefined) {
-            profileStatus = await this.validateProfiles(theProfile);
+            // If the profile has not been validated, and is allowed to be validated, call validateProfiles
+            profileStatus = await this.validateProfiles(theProfile, prompt);
         }
         return profileStatus;
     }
 
-    public async disableValidation(node: IZoweNodeType): Promise<IZoweNodeType>{
+    /**
+     * Handles the validation of a profile by calling getStatus
+     *
+     * @export
+     * @param {boolean} prompt - should the user be prompted for any details missing from the profile?
+     */
+    public async validateProfiles(theProfile: IProfileLoaded, prompt?: boolean) {
+        let filteredProfile: IProfileValidation;
+        let profileStatus;
+        const getSessStatus = await ZoweExplorerApiRegister.getInstance().getCommonApi(theProfile);
+
+        // Filter profilesForValidation to check if the profile is already validated as active
+        this.profilesForValidation.filter((profile) => {
+            if ((profile.name === theProfile.name) && (profile.status === "active")){
+                filteredProfile = {
+                    status: profile.status,
+                    name: profile.name,
+                    session: profile.session
+                };
+            }
+        });
+
+        // If not yet validated or inactive, call getStatus and validate the profile
+        // status will be stored in profilesForValidation
+        if (filteredProfile === undefined) {
+            try {
+                if (getSessStatus.getStatus) {
+                    profileStatus = await vscode.window.withProgress({
+                        location: vscode.ProgressLocation.Notification,
+                        title: localize("Profiles.validateProfiles.validationProgress", "Validating {0} Profile.", theProfile.name),
+                        cancellable: true
+                    }, async (progress, token) => {
+                        token.onCancellationRequested(() => {
+                            // will be returned as undefined
+                            vscode.window.showInformationMessage(
+                                localize("Profiles.validateProfiles.validationCancelled", "Validating {0} was cancelled.", theProfile.name));
+                        });
+                        return getSessStatus.getStatus(theProfile, theProfile.type, prompt);
```

Comment:
z/OSMF and extender profiles return different values for `getSessStatus.getStatus(theProfile, theProfile.type, prompt);`. This might be why extenders are not functioning as expected with validation enabled (e.g. Search and Create don't work), but seem to be able to work when validation is disabled.

Here is an example of what a zosmf profile returns:
<img width="823" alt="Screen Shot 2020-09-15 at 4 48 14 AM" src="https://user-images.githubusercontent.com/45975633/93189765-8b7d3f00-f710-11ea-9e87-86eb1aca6efe.png">

And here is an example of what an extender profile returns:
<img width="823" alt="Screen Shot 2020-09-15 at 4 47 24 AM" src="https://user-images.githubusercontent.com/45975633/93189828-9fc13c00-f710-11ea-932c-fd0c69d41388.png">

(Note I separated the `getStatus` function call into its own line locally, but only so I could better see what was happening during debugging.)